### PR TITLE
Ingress Node Firewall deployment on un-released OCP version (from QE App Registry catalog)

### PR DESCRIPTION
**Pull request by dahorak**
- create QE App Registry catalog source for Ingress Node Firewall
- create also ICSP for Brew registry

### CODE REVIEW

**Comment by dahorak** [comment]
Verification job: https://url.corp.redhat.com/a2d7e05

File: ocs_ci/deployment/ingress_node_firewall.py [hunk 1]
```diff
@@ -106,6 +120,70 @@ class IngressNodeFirewallInstaller(object):
 
     def __init__(self):
         self.namespace = constants.INGRESS_NODE_FIREWALL_NAMESPACE
+        self.source = constants.OPERATOR_CATALOG_SOURCE_NAME
+
+    def icsp_brew_registry_exists(self):
```
Comment (petr-balogh) [inline ocs_ci/deployment/ingress_node_firewall.py:125]: As we discussed offline, I suggest to open issue to move the logic of brew registry to different module and reuse it in other places as well to not repeat the code. This way we can move first code out of this modul to the other, and verify only this change here. The other places - we can replace in separate PRs and verify them separately to know we didn't break other things.

**Review by petr-balogh — DISMISSED** [review]

File: ocs_ci/deployment/ingress_node_firewall.py (hunk 1 above)
Comment (dahorak) [reply ocs_ci/deployment/ingress_node_firewall.py]: For now, I've at least moved the code related to qe-app-registry catalog to new module: 6cad0f41c911dc41a1a14b2c9a6ced49b41897c1

**Comment by dahorak** [comment]
> Also I guess we don't need backport it as lower versions are GAed and we don't need this logic, right? 
yes, no need to backport

**Review by petr-balogh — APPROVED** [review]

**Comment by dahorak** [comment]
Another verification job: https://url.corp.redhat.com/89be95d (this verification includes the last commit).

**Review by fbalak — DISMISSED** [review]
LGTM, icsp() and catalog_source() function names could be more informative (suffix _exists would be good but icsp_brew_registry_exists also exists so it would collide) but it could work as is.

**Review by petr-balogh — APPROVED** [review]

**Pull request merged**